lrpar/src/lib/ctbuilder.rs

```diff
@@ -60,13 +61,19 @@ const RUST_FILE_EXT: &str = "rs";
 const SGRAPH_FILE_EXT: &str = "sgraph";
 const STABLE_FILE_EXT: &str = "stable";
 
+pub enum ActionKind {
+    GenericParseTree,
```

Comment:
We should document these briefly with `///`. Also, should they be in alphabetical order, or sholud we keep `GenericParseTree` first because it's the default? I can be convinced either way!

Reply:
I'm happy with alphabetical.